src/discord/cogs/system.js

```diff
@@ -32,15 +33,24 @@ module.exports = [{
 	name: 'eval',
 	uses: 1,
 	admin: 3,
-	command: (message) => {
+	command: async (message) => {
 		try {
-			let e = eval(message.mss.input); // eslint-disable-line no-eval
+			let e = await eval(message.mss.input); // eslint-disable-line no-eval
```

Comment:
Why would you await eval? Eval is already synchronous and (should) not return any promises. Any that need to return promises should append `.then(()=>{})` to the end of their code, and deal with that appropriately.

Reply:
Well there is no point in doing `.then(v => v)` when the `await` can deal with it properly. Evan can return promises because (for example) when doing `r.table("users").run()`, that returns a promise, so there is no reason to do `.then(v => v)`.